### PR TITLE
Rework the monitoring flag

### PR DESCRIPTION
**Pull request by pypingou**
This replace the two states flag with a three states one:
* True: monitor the package, open a bug and try a scratch build
* False: do not monitor the package
* nobuild: monitor the package, open a bug but do not try a scratch build

This should fix https://github.com/fedora-infra/pkgdb2/issues/187

As a side effect, of the new UI, I believe this will also fix https://github.com/fedora-infra/pkgdb2/issues/163

### CODE REVIEW

**Comment by pypingou** [comment]
The new UI: 

screenshot from 2015-05-19 14 12 32

File: pkgdb2/static/pkgdb.css [hunk 1]
```diff
@@ -156,6 +156,13 @@ a.wheel, a.wheel:hover, button.wheel, button.wheel:hover {
   background-position: 3px 50%;
 }
 
+label.active, label.active:hover {
+  background-image: url(https://admin.fedoraproject.org/pkgdb/static/arrow.png) !important;
```
Comment (ralphbean) [inline pkgdb2/static/pkgdb.css]: Hard-coded?

**Comment by ralphbean** [comment]
:+1: from me (except for that background-image url -- can that be changed?)

File: pkgdb2/static/pkgdb.css (hunk 1 above)
Comment (pypingou) [inline pkgdb2/static/pkgdb.css]: They are hard-coded for all the css, the only other option is to hard-code them as ``/pkgdb/static/...``.

We can't use flask's magic in the CSS files :(

Comment (ralphbean) [inline pkgdb2/static/pkgdb.css]: I could swear you can do them relative to the css file.

Comment (ralphbean) [inline pkgdb2/static/pkgdb.css]: Yeah, see [here how fedmenu does it?](https://github.com/fedora-infra/fedmenu/blob/develop/css/fedmenu.css#L70)

**Comment by pypingou** [comment]
Adjusted :)

**Comment by ralphbean** [comment]
:+1:  Thanks!

**Pull request merged**